packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -617,6 +617,22 @@ taskDefinition.addContainer('TheContainer', {
 });
 ```
 
+When providing the Splunk token in the `token` field it gets resolved to the secret value on deploying. Hence it is encouraged to use the `secretToken` field for specifying the Splunk token as a `Secret` which will be populated in the Log Driver `SecretOptions`. Below is an example task definition with both fields specified:
+
+```ts
+// Create a Task Definition for the container to start
+const taskDefinition = new ecs.Ec2TaskDefinition(this, 'TaskDef');
+taskDefinition.addContainer('TheContainer', {
+  image: ecs.ContainerImage.fromRegistry('example-image'),
+  memoryLimitMiB: 256,
+  logging: ecs.LogDrivers.splunk({
+    token: cdk.SecretValue.secretsManager('my-splunk-token'),
+    secretToken: ecs.Secret.fromSecretsManager(secret),
```

Comment:
I don't think this is a good customer experience now that the customer is recommended to specify both of these fields. They do the same thing, right? 

Since `token` is deprecated, ideally the customer would no longer need to specify the `token` field if they specify `secretToken`. But, `token` can't be made optional because that will break backwards compatibility. 

I'm going to do some research and inquire with other team members to see if there is a better way to do this.

Reply:
Yes, they essentially have the same effect. It would be great to know if the team suggests handling such cases differently.